rs/src/management/http_client.rs

```diff
@@ -456,6 +460,12 @@ impl TunnelManagementClient {
         let headers = request.headers_mut();
         headers.insert("User-Agent", self.user_agent.clone());
 
+        // Add Windows group policies to the header
+        let policy_header_value = self.policy_provider.get_header_value("").await?;
+        if !policy_header_value.is_empty() {
+            headers.insert("Policies", HeaderValue::from_str(&policy_header_value)?);
```

Comment:
nit: I think you can use [`HeaderValue::from_maybe_shared`](https://docs.rs/http/latest/http/header/struct.HeaderValue.html) to avoid copying memory here